site/how-it-works.md

```diff
@@ -95,6 +105,108 @@ the MAC. This way weave peers never need to exchange the MAC addresses
 of clients and need not take any special action for ARP traffic and
 MAC discovery.
 
+### <a name="topology"></a>Topology Communication
+
+Topology is communicated over the TCP links between peers.
+There are two message types:
+
+##### FetchAll
+This carries no payload. The receiver responds with the entire
+topology model as the receiver has it. FetchAll is sent:
+
+ * when a new connection is established, by the peer that initiated the connection
```

Comment:
I don't see the point. The doc is meant to document the state as of now. Which includes #349. As well as a whole bunch of changes over the history of the project.